packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/list/routing.js

```diff
@@ -0,0 +1,50 @@
+import { TRACKING_NAME } from '../../../../constants';
+import { TRACKING_SUFFIX as LISTENERS_TACKING_SUFFIX } from '../../../constants';
+import { TRACKING_SUFFIX as L7_TACKING_SUFFIX } from '../constants';
+import { TRACKING_SUFFIX } from './constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'octavia-load-balancer.loadbalancer.listeners.listener.l7Policies.list',
+    {
+      url: '/list',
+      views: {
+        loadbalancerL7View: 'octaviaLoadBalancerL7PoliciesList',
+      },
+      resolve: {
+        breadcrumb: () => null,
+        policies: /* @ngInject */ (
+          OctaviaLoadBalancerL7Service,
+          projectId,
+          region,
+          listenerId,
+        ) =>
+          OctaviaLoadBalancerL7Service.getPolicies(
+            projectId,
+            region,
+            listenerId,
+          ),
+        goToL7PolicyCreation: /* @ngInject */ ($state, trackL7Action) => () => {
+          trackL7Action('create');
```

Comment:
as you're importing constant file, you can is it to define a constant `'create'`

Reply:
For this one, I think it's better to do it in the policy creation ticket, as there I have created the constant